src/averageAges.js

```diff
@@ -14,7 +14,21 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
+  const men = people.filter(elem => {
+    if (century) {
+      return (elem.sex === 'm' && Math.ceil(elem.died / 100) === century);
+    }
+
+    return elem.sex === 'm';
+  });
+
+  const summAge = men.reduce((prev, current, index) => {
```

Comment:
why do u need index here?

move logic of looking for average to separate function, to use it in all functions